docs/release_notes.md

```diff
@@ -1,5 +1,69 @@
 # Release Notes
 
+## Version 1.1.0
+
+### Added
+
+#### Core
+
+- Shift workflows now run on a dedicated thread, enabling users to inspect and author other workflows and interact with the UI while there is an ongoing execution.
+- An environment variable, `SHIFT_SINGLE_THREADED`, has been added to enable or disable threaded execution.
```

Comment:
Question, here we're mentioning the use of the environment variable, should we also mention the option in the menu to change it on-the-fly?

Reply:
yes, let me do it.

Reply:
done